src/Directory.Build.targets

```diff
@@ -54,4 +54,43 @@
   <Target Name="DumpTargets" BeforeTargets="ResolveProjectReferences">
     <Message Text="DumpTargets> $(OutputPath), C=[$(Configuration)], CG=[$(ConfigurationGroup)], OG=[$(OSGroup)], TG=[$(TargetGroup)]" Importance="Low" />
   </Target>
+
+  <PropertyGroup Condition="'$(IsInvokableTestProject)' == 'true'">
+    <LaunchSettingsOutputFileFullPath Condition="'$(LaunchSettingsOutputFileFullPath)' == ''">$(AppDesignerFolder)/launchSettings.json</LaunchSettingsOutputFileFullPath>
+    <LaunchSettingsInputFileFullPath Condition="'$(LaunchSettingsInputFileFullPath)' == ''">$(ToolsDir)tests.targets</LaunchSettingsInputFileFullPath>
+
+    <CompileDependsOn>
+      GenerateLaunchSettingsFile;
+      $(CompileDependsOn);
+    </CompileDependsOn>
+  </PropertyGroup>
+
+  <!-- Create debug support files -->
+  <Target Name="GenerateLaunchSettingsFile"
+          Condition="'$(LaunchSettingsOutputFileFullPath)' != '' AND '$(OmitLaunchSettings)' != 'true'"
+          Inputs="$(LaunchSettingsInputFileFullPath)"
+          Outputs="$(LaunchSettingsOutputFileFullPath)">
+
+    <PropertyGroup>
+      <LaunchSettingsFileContent>
+{
+  &quot;profiles&quot;: {
+    &quot;xUnit.net Console&quot;: {
+      &quot;commandName&quot;: &quot;Executable&quot;,
+      &quot;executablePath&quot;: &quot;$(StartProgram.Replace('\', '\\'))&quot;,
+      &quot;commandLineArgs&quot;: &quot;$(StartArguments)&quot;,
+      &quot;workingDirectory&quot;: &quot;$(StartWorkingDirectory.Replace('\', '\\'))&quot;
+    }
+  }
+}
+      </LaunchSettingsFileContent>
+    </PropertyGroup>
+
+    <MakeDir Directories="$(AppDesignerFolder)" />
```

Comment:
It would be more correct if we made the folder that `$(LaunchSettingsOutputFileFullPath)` is pointing to. Here we assume it is pointing to `$(AppDesignerFolder)`, but above we allow it to point somewhere else (since we condition on it not being set already)